citar.el

```diff
@@ -885,7 +896,7 @@ into the corresponding reference key.  Return
 
 ;;;###autoload
 (defun citar-open (keys-entries)
-  "Open related resources (links or files) for KEYS-ENTRIES."
+  "Open any related resources (links, files, or notes) for KEYS-ENTRIES."
   (interactive (list (citar-select-refs
```

Comment:
isn't this still CRM?

Reply:
Only if `citar-select-multiple` is set to `t`, I think?